app-modules/meeting-center/src/Providers/MeetingCenterServiceProvider.php

```diff
@@ -26,6 +28,16 @@ public function boot(): void
             'calendar_event' => CalendarEvent::class,
         ]);
 
+        $this->callAfterResolving(Schedule::class, function (Schedule $schedule) {
```

Comment:
I think we will want to add a `SyncCalendars` job to be the scheduled entity, and dispatch the `SyncCalendar` jobs from there. I worry about the number of scheduled jobs we are adding here as the number of calendars grows.